tests/test_wave_01.py

```diff
@@ -184,7 +184,7 @@ def test_watch_movie_moves_movie_from_watchlist_to_empty_watched():
     assert updated_data["watched"][0]["genre"] == "Fantasy"
     assert updated_data["watched"][0]["rating"] == 4.8
 
```

Comment:
This file didn't need to be pushed up. I recommend using `git add <filename>` to specify which file(s) and changes you'd like to record and push up to github.